From: Bradley Morgan <brads@mainlining.org>
Date: Tue, 25 Aug 2026 15:11:57 +0000
Subject: [PATCH] mailmap: update email address for Bradley Morgan

I switched to brads@mainlining.org for kernel work, so map the old
include@grrlz.net address over to keep shortlog and blame from splitting
commits between the two.

Link: https://lore.kernel.org/20260825151157.4533-1-brads@mainlining.org
Signed-off-by: Bradley Morgan <brads@mainlining.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---
 .mailmap | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.mailmap b/.mailmap
index 0d4e9bc69ebe8..1b47441b2b224 100644
--- a/.mailmap
+++ b/.mailmap
@@ -170,6 +170,7 @@ Boris Brezillon <bbrezillon@kernel.org> <b.brezillon.dev@gmail.com>
 Boris Brezillon <bbrezillon@kernel.org> <b.brezillon@overkiz.com>
 Boris Brezillon <bbrezillon@kernel.org> <boris.brezillon@bootlin.com>
 Boris Brezillon <bbrezillon@kernel.org> <boris.brezillon@free-electrons.com>
+Bradley Morgan <brads@mainlining.org> <include@grrlz.net>
 Brendan Higgins <brendan.higgins@linux.dev> <brendanhiggins@google.com>
 Brendan Jackman <brendan.jackman@linux.dev> <jackmanb@google.com>
 Brian Avery <b.avery@hp.com>
-- 
2.47.3

